### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode characters in position 332-333: ordinal not in range(128)

### ISSUE THREAD

**Issue by qiuhongxiang** [TylerYep/torchinfo#96]
**Describe the bug**
when I use the plug-in，it will report an error: 
Traceback (most recent call last):
  File "test.py", line 36, in <module>
    model_stats = summary(model, (1, 3, 112, 112), col_names=["input_size", "output_size"])
  File "/usr/local/lib/python3.6/dist-packages/torchinfo/torchinfo.py", line 199, in summary
    print(results)
I want to underst the problem! thanks.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by TylerYep** [comment]
Please post the full error message you are getting,  as well as the source code you are using

**Comment by qiuhongxiang** [comment]
> Please post the full error message you are getting, as well as the source code you are using

source code:

import torch
from torchinfo import summary
import torchvision.models as models 

if __name__ == '__main__':
    print(torch.__version__)

    model = models.alexnet()
    model_stats = summary(model, (1, 3, 112, 112), col_names=["input_size", "output_size"])

question:
1.10.0+cu113
Traceback (most recent call last):
  File "tmp.py", line 12, in <module>
    model_stats = summary(model, (1, 3, 112, 112), col_names=["input_size", "output_size"])
  File "/usr/local/lib/python3.6/dist-packages/torchinfo/torchinfo.py", line 199, in summary
    print(results)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 332-333: ordinal not in range(128)

thanks

**Comment by TylerYep** [comment]
This looks like you are using a machine or Python interpreter that doesn't have access to unicode.

I can put out a fix for this. Some workarounds in the meantime are to:
```
output = summary(model, ..., verbose=0)
print(output.encode('ascii', 'ignore').decode('ascii'))
```
or
- find the source code of your installed torchinfo and remove all of the unicode characters in the files.

Once this is fixed, you should be able to use:
```
summary(model, ..., row_settings=("depth", "ascii"))
```

**Comment by qiuhongxiang** [comment]
thans

**Comment by ahrnbom** [comment]
I can confirm that my Python install does support UTF-8 and I still had to do that trick, slightly modified (in case anyone else finds this useful):

```
output = summary(net, inp, verbose=0)
text = str(output).encode('ascii', 'ignore').decode('ascii')
print(text)
```

**Comment by TylerYep** [comment]
Hi everyone, this feature was added in 2b82275 and will be included in v1.6.1. For situations in which non-ascii characters are invalid, you can use:
```
summary(model, ..., row_settings=("depth", "ascii_only"))
```
